lib/coinbase/faucet_transaction.rb

```diff
@@ -17,5 +19,17 @@ def initialize(model)
     def transaction_hash
       model.transaction_hash
     end
+
+    # Returns a String representation of the FaucetTransaction.
+    # @return [String] a String representation of the FaucetTransaction
+    def to_s
+      "Coinbase::FaucetTransaction{transaction_hash: '#{transaction_hash}'}"
```

Comment:
should we add the blockexplorer link to help? 
```
https://sepolia.basescan.org/tx/#{transaction_hash}
```

Reply:
Done